package.json

```diff
@@ -2,11 +2,19 @@
   "name": "remeda",
   "version": "0.0.35",
   "description": "A utility library for JavaScript and Typescript.",
-  "main": "dist/commonjs/index.js",
-  "module": "dist/es/index.js",
-  "jsnext:main": "dist/es/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "jsnext:main": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.mts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
   "sideEffects": false,
   "devDependencies": {
+    "@swc/core": "^1.3.78",
```

Comment:
Why do we include swc? I thought tsup uses esbuild under the hood.

Reply:
I did this because I got an error saying that I needed it for ES5.